libs/libs/integrations/twilio.star

```diff
@@ -0,0 +1,65 @@
+_BASE_URL = "https://api.twilio.com/2010-04-01"
+
+def connect(conn, account_sid, auth_token=None):
```

Comment:
This is a python reference for twilio sdk, I think we should stick with this standard instead of the connect option (rename connect to Client or NewClient)
```python
from twilio.rest import Client

# Your Account SID and Auth Token from console.twilio.com
account_sid = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX"
auth_token  = "your_auth_token"

client = Client(account_sid, auth_token)

message = client.messages.create(
    to="+15558675309",
    from_="+15017250604",
    body="Hello from Python!")

print(message.sid)
```